mkosi.md

```diff
@@ -1082,6 +1056,123 @@ images. Any distribution that packages `zypper` may be used to build
 
 Currently, *Fedora Linux* packages all relevant tools as of Fedora 28.
 
+# SCRIPTS
```

Comment:
I think it would be helpful to move the execution flow section down here, so people can easily compare it with the scripts.